README.md

```diff
@@ -122,6 +122,15 @@ is a representation of the entity producing telemetry.
 |--------------------------|--------------------------|------------------------------------------------------------------------------|
 | otel.resource.attributes | OTEL_RESOURCE_ATTRIBUTES | Used to specify resource attributes in format: key1=val1,key2=val2,key3=val3 |
 
+#### Peer service name
+
+The [peer service name](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/span-general.md#general-remote-service-attributes) is
+the name of a remote service that is being connected to. It roughly corresponds to `service.name` in the [Resource](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#service) for the local service.
```

Comment:
why `roughly` and not `exactly`? You want to correlate them, don't you?

Reply:
Good point not sure why I took a conservative approach there